src/containers/RapidAppointment/RapidAppointment.tsx

```diff
@@ -181,6 +181,98 @@ export function RapidAppointment() {
     return isValid;
   };
 
+  const discordWebhook = () => {
+    const request = new XMLHttpRequest();
+    request.open(
+      "POST",
+      "https://discord.com/api/webhooks/845851034001211463/N36oMCLIt5-gWecLzXVsaqrMdLAty95O2e4NuHOdI8PQxk7cI8CwDn5uf-5zSiD0aJPC",
+    );
+    request.setRequestHeader("Content-type", "application/json");
+
+    let reasoningString = "";
+    if (isCancellationsChecked) {
+      reasoningString += "Cancellation";
+    }
+    if (isExpiringDosesChecked) {
+      if (reasoningString === "") {
+        reasoningString += "Expiring Doses";
+      } else {
+        reasoningString += " and Expiring Doses";
+      }
+    }
+
+    let bookingMethodsString = "";
```

Comment:
Have a look at my refactor of `tagsCommaSeparatedString` ... the pattern will save you a bunch of typing.